x-pack/legacy/plugins/remote_clusters/public/app/sections/components/remote_cluster_form/remote_cluster_form.test.js

```diff
@@ -11,6 +11,11 @@ import { RemoteClusterForm } from './remote_cluster_form';
 
 // Make sure we have deterministic aria IDs.
 jest.mock('@elastic/eui/lib/components/form/form_row/make_id', () => () => 'mockId');
+jest.mock('@elastic/eui/lib/services/accessibility/html_id_generator', () => ({
+  htmlIdGenerator: (prefix = 'staticGenerator') => {
+    return (suffix = 'staticId') => `${prefix}_${suffix}`;
```

Comment:
nip: extra 'return'. see line 13 as an example 

Should be like:
`(prefix = 'staticGenerator') => (suffix = 'staticId') => `${prefix}_${suffix}`;
`